blogs/posts/2023-04-26_alternative_remotes.qmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Alternative remote repositories"
+author: "Tom Jemmett"
+date: "2023-04-26"
+categories: [git, tutorial]
+knitr:
+  opts_chunk:
+    fig.height: 2.5
+    echo: false
+    dev: png
+    dev.args:
+      bg: transparent
+---
+
+It's great when someone send's you a pull request on GitHub to fix bugs or add new features to your project, but you probably always want to check the other persons work in someway before merging that pull request.
+
+All of the steps below are intended to be entered via a terminal.
+
+Let's imagine that we have a GitHub account called example and a repository called test, and we use https rather than ssh.
+
+```
+$ git remote get-url origin
```

Comment:
I stupidly copied this from GitHub's clone/fork button thing and then tried to type `git remote get-https://github.com/The-Strategy-Unit/data_science.git origin` which of course didn't work!